nodeup/pkg/model/context.go

```diff
@@ -693,3 +693,7 @@ func (c *NodeupModelContext) CNIBinDir() string {
 func (c *NodeupModelContext) CNIConfDir() string {
 	return "/etc/cni/net.d/"
 }
+
+func (c *NodeupModelContext) InstallNvidiaRuntime() bool {
+	return c.NodeupConfig.Nvidia != nil && fi.BoolValue(c.NodeupConfig.Nvidia.Install)
```

Comment:
I think in most cases we assume true if Enable / Install is nil.
```suggestion
	return c.NodeupConfig.Nvidia != nil && (c.NodeupConfig.Nvidia.Install == nil || fi.BoolValue(c.NodeupConfig.Nvidia.Install))
```

Reply:
Scanning `bootstrapchannelbuilder.go` we don't assume that for most addons.